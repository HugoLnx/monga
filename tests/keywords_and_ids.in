char character int integer float floatvar void voidfunc
if ifx else elsex while whilex new newx return returnx
_99 _
