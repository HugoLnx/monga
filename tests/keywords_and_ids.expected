TK_CHAR,
TK_ID,character
TK_INT,
TK_ID,integer
TK_FLOAT,
TK_ID,floatvar
TK_VOID,
TK_ID,voidfunc
TK_IF,
TK_ID,ifx
TK_ELSE,
TK_ID,elsex
TK_WHILE,
TK_ID,whilex
TK_NEW,
TK_ID,newx
TK_RETURN,
TK_ID,returnx
TK_ID,_99
TK_ID,_
